### DISCUSSION TITLE
Will dify-sandbox be open source?

### DISCUSSION THREAD

**Discussion by clyfish** [langgenius/dify#3279] [Help] [answered]
After upgrading to 0.6.0, there is a new service [dify-sandbox](https://hub.docker.com/r/langgenius/dify-sandbox), but I didn't find any documentation for it.

Will dify-sandbox be open source?

**Comment by takatost** [comment]
Will open source it when we think the time is right.

**Reply by SSebo** [reply, marked as answer]
I found sandbox opened here https://github.com/langgenius/dify-sandbox

**Comment by mkaskov** [comment]
sandbox has a lot of problems and works good only with simplest cases. It could be great if we can look inside and try to understand it.

**Reply by SSebo** [reply]
I found sandbox opened here https://github.com/langgenius/dify-sandbox
